logback-contrib/pom.xml

```
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2024, LINAGORA. All rights reserved.
+
+    This file is licensed under the terms of the
+    Eclipse Public License v1.0 as published by
+    the Eclipse Foundation.
+
+    This work is based on the work found at
+    https://github.com/qos-ch/logback-contrib
+    authored by the logback-contrib developers.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.linagora.tmail</groupId>
+        <artifactId>tmail-backend-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>com.linagora.tmail</groupId>
+    <artifactId>logback-contrib-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Logback Contrib</name>
+    <url>https://github.com/qos-ch/logback-contrib/wiki</url>
+    <description>
+        Logback Contrib provides community-driven and supported
+        extensions to the Logback logging framework. All contributions
+        are made by contributors with CLA on file.
+    </description>
+    <inceptionYear>2012</inceptionYear>
+
+   <licenses>
+     <license>
+       <name>Eclipse Public License - v 1.0</name>
+       <url>http://www.eclipse.org/legal/epl-v10.html</url>
+     </license>
+   </licenses>
+
+    <developers>
+        <developer>
+            <id>lhazlewood</id>
+            <name>Les Hazlewood</name>
+            <email>les@hazlewood.com</email>
+            <url>http://www.leshazlewood.com</url>
+            <timezone>-8</timezone>
+        </developer>
+        <developer>
+            <id>belaso</id>
+            <name>Christian Trutz</name>
+            <email>christian.trutz@belaso.de</email>
+            <url>http://www.belaso.de</url>
+            <timezone>+2</timezone>
+        </developer>
+        <developer>
+            <id>tony19</id>
+            <name>Tony Trinh</name>
+            <email>tony19@gmail.com</email>
+            <url>http://tony19.github.com</url>
+            <timezone>-5</timezone>
+        </developer>
+    </developers>
+    <contributors>
+    	<contributor>
+    		<name>David J. M. Karlsen</name>
+    		<email>david@davidkarlsen.com</email>
+    	</contributor>
```

```suggestion
            <name>David J. M. Karlsen</name>
    	    <email>david@davidkarlsen.com</email>
    	</contributor>
```

TAB shall be replaced by 4 spaces